SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/GridBasedLBStrategy.cpp

```diff
@@ -157,6 +163,26 @@ SpatialGDK::QueryConstraint UGridBasedLBStrategy::GetWorkerInterestQueryConstrai
 	return Constraint;
 }
 
+FVector2D UGridBasedLBStrategy::GetActorLoadBalancingPosition(const AActor& Actor) const
+{
+	USpatialNetDriver* NetDriver = Cast<USpatialNetDriver>(Actor.GetNetDriver());
+	const Worker_EntityId ActorEntityId = NetDriver->PackageMap->GetEntityIdFromObject(&Actor);
```

Comment:
We could go for this change, but I'm not a fan since this implies that the LB strat is working properly without a PackageMap which doesn't seem to be true. Otherwise, could change/remove the offending test.
```suggestion
	const Worker_EntityId ActorEntityId = IsValid(NetDriver->PackageMap) ? NetDriver->PackageMap->GetEntityIdFromObject(&Actor) : SpatialConstants::INVALID_ENTITY_ID;
```

Reply:
Not an issue after removing Spatial Position change.